script/Arcueid/Day 1/01_00_ARC01_2_1.txt

```diff
@@ -261,7 +261,7 @@ As a child, I couldn't help but be bored by the lifestyle inside that mansion.
 [sha:52d80acb665d5849f793aa68b2b76cbb7aac5681]{
 -- Page 154, Offset 1561.
 -- だから“有間の家に預ける”と言われた時、さして抵抗もなく家を出ることを受け入れた。
-That's why, when I was told that I would be left at the Arima house, I accepted without much resistance.
+So when I was told that I would be left at the Arimas, I accepted without much resistance.
```

Comment:
Prefer `that's why`

Reply:
coming from "As a child, ..." I prefer "So".